packages/neutrino/bin/neutrino

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env node
 
+// TODO: Remove this once babel-loader updates
+// https://github.com/babel/babel-loader/pull/391
+process.noDeprecation = true;
+
 const Neutrino = require('../src/neutrino');
 const yargs = require('yargs');
-const path = require('path');
-const pkg = require(path.join(process.cwd(), 'package.json'));
+const { pathOr } = require('ramda');
+const { join } = require('path');
+
+const cwd = process.cwd();
+let pkg = {};
+
+try {
+  pkg = require(join(cwd, 'package.json'));
+} catch (ex) {}
```

Comment:
No, we try to load data from package.json if it exists, but it's not going to affect anything if we can't find it.